examples/compute-management/02-VirtualMachineCreation.sh

```diff
@@ -3,15 +3,15 @@ set -e
 printf "\n=== Managing Virtual Machine Creation in Azure Compute ===\n"
 
 printf "\n1. Creating a new resource group: %s and location: %s.\n" "$groupName" "$location"
-az group create -n "$groupName" --location "$location"
+az resourcemanager group create -n "$groupName" --location "$location"
```

Comment:
This is the most used cmdlet in the entire set opf cmdlets - you have to create a resource group to do anything else.  Do we really want the cmdlet name to be so wordy?